ScreenSelectorExample/ScreenSelectorExampleProject/Assets/ClosingScript.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+public class ClosingScript : MonoBehaviour
+{
+    private void OnApplicationQuit()
+    {
+        List<string> settings = new List<string>
+            {
+                "Screenmanager Resolution Width",
+                "Screenmanager Resolution Height",
+                "Screenmanager Fullscreen mode",
+                "Screenmanager Stereo 3D",
+                "UnityGraphicsQuality",
+                "UnitySelectMonitor"
+            };
+
+        using (StreamWriter file = new StreamWriter("ScreenSelectorPrefs.txt"))
```

Comment:
Should probably save it somewhere like AppData folder, as the app might be installed to non-writeable location (like Program Files).